app/test/service/openid/github_actions_id_token_test.dart

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+@Tags(['presubmit-only'])
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:http/http.dart' as http;
+import 'package:pub_dev/service/openid/github_openid.dart';
+import 'package:pub_dev/service/openid/jwt.dart';
+import 'package:test/test.dart';
+
+import '../../shared/test_services.dart';
+
+void main() {
+  final actionsIdTokenRequestUrl =
+      Platform.environment['ACTIONS_ID_TOKEN_REQUEST_URL'] ?? '';
+  final actionsIdTokenRequestToken =
+      Platform.environment['ACTIONS_ID_TOKEN_REQUEST_TOKEN'] ?? '';
+  testWithProfile(
+    'id-token test',
+    skip: actionsIdTokenRequestUrl.isEmpty,
+    fn: () async {
+      expect(actionsIdTokenRequestUrl, isNotEmpty);
+      expect(actionsIdTokenRequestToken, isNotEmpty);
+
+      final uri = Uri.parse(actionsIdTokenRequestUrl);
+      final rs = await http.get(
+        uri.replace(
+          queryParameters: {
+            ...uri.queryParameters,
+            'audience': 'https://pub.dev',
```

Comment:
Can we use a dummy audience? so we don't leak something that could potentially be used.